website/src/controllers/api-v3/quests.js

```diff
@@ -65,4 +65,58 @@ api.inviteToQuest = {
   },
 };
 
+/**
+ * @api {post} /groups/:groupId/quests/reject Reject a quest
+ * @apiVersion 3.0.0
+ * @apiName RejectQuest
+ * @apiGroup Group
+ *
+ * @apiParam {string} groupId The group _id (or 'party')
+ * @apiParam {string} questKey The quest _id
+ *
+ * @apiSuccess {Object} Quest Object
+ */
+api.rejectQuest = {
+  method: 'POST',
+  url: '/groups/:groupId/quests/reject',
+  middlewares: [authWithHeaders(), cron],
+  async handler (req, res) {
+    let user = res.locals.user;
+    let questKey = user.party.quest.key;
+    let quest = questScrolls[questKey];
+
+    req.checkParams('groupId', res.t('groupIdRequired')).notEmpty();
+
+    let validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    let group = await Group.getGroup({user, groupId: req.params.groupId, fields: 'type quest'});
+    if (!group) throw new NotFound(res.t('groupNotFound'));
+    if (!quest) throw new NotFound(res.t('questNotFound', { key: questKey }));
+    if (!group.quest.key) throw new NotFound(res.t('questInvitationDoesNotExist'));
+
+    let analyticsData = {
+      category: 'behavior',
+      owner: false,
+      response: 'reject',
+      gaLabel: 'reject',
+      questName: group.quest.key,
+      uuid: user._id,
+    };
+    track('quest', analyticsData);
+
+    group.quest.members[user._id] = false;
+    group.markModified('quest.members');
+    await group.save();
+
+    user.party.quest.RSVPNeeded = false;
+    user.party.quest.key = null;
+    await user.save();
```

Comment:
The user and group save calls should probably be done together, since they can be parallelized.

```
let [savedGroup] = await Q.all([
  group.save(),
  user.save(),
]);

res.respond(200, savedGroup.quest);
```